kvmagent/kvmagent/plugins/network_plugin.py

```
@@ -600,6 +771,276 @@ def delete_bonding(self, req):
+        return interface_lldp_info
+
+    @linux.retry_if_unexpected_value(None, times=5, sleep_time=5)
+    def _get_interface_lldp(self, interface_name):
+        lldpinfo = None
+        r, info = bash_ro('lldpctl ports %s -f json' % interface_name)
+        if r != 0:
+            return lldpinfo
+
+        try:
+            lldpinfo = self._parse_lldp_json(info, interface_name)
+        except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)